src/main/java/frc/robot/Robot.java

```diff
@@ -13,14 +13,26 @@
 
 package frc.robot;
 
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
```

Comment:
We can revert all of the changes in this file. As is, we'll constantly be scanning for April Tags when the robot starts up, but we disabled the command scheduler, so none of our other control logic will actually execute